msg/tools/generate_microRTPS_bridge.py

```diff
@@ -250,18 +250,21 @@ def check_rtps_id_uniqueness(classifier):
         "FastRTPSGen not found. Specify the location of fastrtpsgen with the -f flag")
 
 # get ROS 2 version, if exists
-ros2_distro = ""
-try:
-    rosversion_out = subprocess.check_output(["rosversion", "-d"])
-    rosversion_out = rosversion_out.rstrip()
-    if rosversion_out not in ["<unknown>", "kinetic", "lunar", "melodic"]:
-        ros2_distro = rosversion_out
-except OSError as e:
-    if e.errno == errno.ENOENT:
-        if args.ros2_distro != None:
-            ros2_distro = args.ros2_distro
-    else:
-        raise
+ros2_distro = os.environ.get('ROS_DISTRO')
```

Comment:
Can you please add a check that makes sure that makes sure that the ROS_DISTRO that is set is not a ROS 1 distro? Can be done on the next line. Otherwise, the rest of the process will fail. Thanks.